nemo/collections/nlp/modules/common/megatron/adapters/mcore_mixins.py

```diff
@@ -13,14 +13,20 @@
 # limitations under the License.
 
 import torch
+import torch.nn.functional as F
+from megatron.core.fusions.fused_bias_dropout import get_bias_dropout_add
+from megatron.core.fusions.fused_bias_gelu import bias_gelu_impl
 from megatron.core.models.gpt.gpt_embedding import GPTEmbedding
 from megatron.core.transformer.attention import SelfAttention
+from megatron.core.transformer.mlp import MLP
 from megatron.core.transformer.transformer_layer import TransformerLayer
 from megatron.core.utils import make_viewless_tensor
```

Comment:
I think you need to import guard megatron.core. Can see how we do this in megatron_gpt_model.py